src/styles/styleUtils.js

```diff
@@ -0,0 +1,8 @@
+import { css } from 'styled-components'
+
+export const transitionCss = css`
+    transition: all .5s ease-out;
+    -moz-transition: all .5s ease-out;
+    -webkit-transition: all .5s ease-out;
+    -o-transition: all .5s ease-out;
+`
```

Comment:
this might be a conflict with my file but idk yet, just keep that in mind

Reply:
i've merged your changes and it's identical so no conflicts yay